lib/api/environment_user_access_api.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:client_common/api/lenra_http_client.dart';
```

Comment:
Rename this file to follow the name used in the `server`

environment_user_access_api.dart -> user_environment_access_api.dart